README.md

```diff
@@ -43,8 +42,12 @@ tox -e tests
 ```
 
 ### Documentation
```

Comment:
We may want to use only two `#` here, so that `Requirements` is nested inside `Documentation` and `Documentation` is outside of `Testing`.